forms-flow-service/src/resourceBundles/zh/resourceBundles.ts

```diff
@@ -531,7 +531,7 @@ export const RESOURCE_BUNDLES_ZH = {
   "Your form is now ready!": "您的表单现在已准备好！",
   "Please tell me all the specifications and requirements you have in mind for your form. After you have finished the description, hit send.": "请告诉我您对表单的所有规格和要求。完成描述后，请按发送。",
   "Choose template": "选择模板",
-  "Search by bundle name..": "按包名称搜索..",
+  "Search by bundle name": "按包名称搜索..",
```

Comment:
The translated text at the end still contains double dots that haven't been removed. can you please check
![image](https://github.com/AOT-Technologies/forms-flow-ai-micro-front-ends/assets/99173163/7c194c08-4505-48ed-abb5-6d98ff07a129)

Reply:
updated